content/news/vcmi-0.100-release.md

```diff
@@ -0,0 +1,35 @@
++++
+categories = ["Releases", "Stable"]
+date = "2022-09-10"
+description = ""
+featured = "vcminewterrain.jpg"
+featuredalt = ""
+featuredpath = "date"
+linktitle = ""
+title = "VCMI 0.100 released"
+type = "post"
+forumurl = "https://forum.vcmi.eu/"
+
++++
+
+## We're happy to announce new release!
+
+Few years passed after we released VCMI 0.99 and now it's time for a new release.
+Tons of changes, improvements, and fixes have been introduces during that time, it's even hard to recall all of them.
+However we'd like to highlight several major changes.
+
```

Comment:
## Game engine

Lodestar grail finally can reveal all the map!
We were also working hard to track down and fix the most of the crashes constantly hurting VCMI gameplay. Now they are rare beasts but if you manage to find one do not hesitate to report it!

Reply:
I also think if we should mention our new buttons on hero exchange window. Something like
## Adventure map

Hero exchange window has got a few additional buttons and keyboard shortcuts to ease army and artifact exchange. If you do not see them after installing the release ensure you have latest VCMI essentials mod.